readthedocs/settings/base.py

```diff
@@ -763,3 +763,12 @@ def DOCKER_LIMITS(self):
             },
         },
     }
+
+    RTD_EMBED_API_EXTERNAL_DOMAINS = [
```

Comment:
I feel like this should probably be in the DB, instead of a setting. I guess we can start with this for now, but I wonder about adding something like https://github.com/zlorf/django-dbsettings/ in the future to make it easier to maintain this list.

Reply:
The ability to change settings on the fly has it's pros and cons. In particular, is hard to track the changes and it's easier to break production by mistake.

I don't really think we will need to change this too frequently. I'm fine starting with a Django setting for now. We could also create a `EmbedExternalDomain` or similar to add these domains on the fly via the admin if needed instead of opening the door to something bigger like db-settings.

Reply:
> The ability to change settings on the fly has it's pros and cons. In particular, is hard to track the changes and it's easier to break production by mistake.

Good point. Let's keep it simple for now -- but requiring a full deploy to change these could get annoying, but we don't need to solve that problem yet. 

> I don't really think we will need to change this too frequently. I'm fine starting with a Django setting for now. We could also create a EmbedExternalDomain or similar to add these domains on the fly via the admin if needed instead of opening the door to something bigger like db-settings.

Yea, seems like just re-implementing a dbsettings with more work on our side :) I think there's probably 5-10 places where this could be really useful, along with model change tracking, so we can audit the changes over time.

Reply:
I created an issue to track and discuss this in a more general way.

> along with model change tracking, so we can audit the changes over time.

👍🏼